### PR TITLE
pgmodeler: 0.9.2 -> 0.9.3

### PR DESCRIPTION
**Pull request by 1000101**
###### Motivation for this change
Upgrade pgModeler to the latest version. This version also fixes crashes after startup present in the current version.

Note: Database model file structure has changed - this can affect anyone upgrading from earlier versions. More info [here](https://pgmodeler.io/blog/2020/12/30/pgmodeler-093-is-finally-here-after-a-long-year-of-hard-work).

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by esclear — APPROVED** [review]
Looks good to me.

File: pkgs/applications/misc/pgmodeler/default.nix [hunk 1]
```diff
@@ -23,7 +30,6 @@ mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "A database modeling tool for PostgreSQL";
-    longDescription = ''pgModeler (PostgreSQL Database Modeler) is an open source database modeling tool designed for PostgreSQL.'';
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/pgmodeler/default.nix:26]: Why did you remove the long description?

Comment (1000101) [reply pkgs/applications/misc/pgmodeler/default.nix:26]: It seemed a bit redundant to me (rephrased description).

File: pkgs/applications/misc/pgmodeler/default.nix [hunk 2]
```diff
@@ -1,16 +1,23 @@
-{ stdenv, lib, fetchFromGitHub, pkgconfig, qmake, mkDerivation,
-  qtsvg,
-  libxml2, postgresql }:
+{ stdenv
+, lib
+, fetchFromGitHub
+, pkgconfig
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/pgmodeler/default.nix:4]:
```suggestion
, pkg-config
```

File: pkgs/applications/misc/pgmodeler/default.nix [hunk 3]
```diff
@@ -1,16 +1,23 @@
...
 
   src = fetchFromGitHub {
     owner = "pgmodeler";
     repo = "pgmodeler";
     rev = "v${version}";
-    sha256 = "1wkvg20krfwkziz7skgmwlinx07jm5nl3455payg5brv69zf60kl";
+    sha256 = "1bci5x418dbnkny7hn0b5q5lxyajrgl3frv41ji0hcw9vivrds2g";
   };
 
   enableParallelBuilding = true;
```
Comment (SuperSandro2000) [inline pkgs/applications/misc/pgmodeler/default.nix:23]:
```suggestion
```

set by qmake.

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/pgmodeler/default.nix (hunk 1 above)
Comment (1000101) [reply pkgs/applications/misc/pgmodeler/default.nix]: I've reverted it, let me know if you changed your mind.

**Comment by 1000101** [comment]
@GrahamcOfBorg build pgmodeler

File: pkgs/applications/misc/pgmodeler/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/applications/misc/pgmodeler/default.nix]: On a second look it is almost the same. keep it.

Comment (1000101) [reply pkgs/applications/misc/pgmodeler/default.nix:26]: Ok, I've force pushed the change.

**Comment by 1000101** [comment]
@GrahamcOfBorg build pgmodeler

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review which does not build all packages (e.g. lumo, tensorflow or pytorch)
If you find some bugs or got suggestions for further things to search or run please reach out to SuperSandro2000 on IRC.

Result of `nixpkgs-review pr 108463` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
1 package built:
</summary>
<ul>
<li>pgmodeler</li>
</ul>
</details>
